core/templates/services/UpgradedServices.ts

```diff
@@ -608,6 +610,9 @@ export class UpgradedServices {
       new AudioTranslationLanguageService(
         upgradedServices['BrowserCheckerService'],
         upgradedServices['LanguageUtilService']);
+    upgradedServices['CollectionCreationBackendService'] =
```

Comment:
Please wrap in `(..)` here and break after `(`.

Reply:
I didn't get what's expected here? Could you please explain that again?

Reply:
Sure, I meant do this:
```suggestion
    upgradedServices['CollectionCreationBackendService'] = (
      new CollectionCreationBackendService(
        upgradedServices['HttpClient']));
```

Reply:
Is this expected? I don't see such pattern across the file.

Reply:
Yes, please see the style guideline [here](https://github.com/oppia/oppia/wiki/Coding-style-guide#javascript).